com.eclipsesource.modelserver.client/src/main/java/com/eclipsesource/modelserver/client/ModelServerClient.java

```diff
@@ -93,10 +98,12 @@ public ModelServerClient(OkHttpClient client, String baseUrl) throws MalformedUR
     }
 
     @Override
-    public CompletableFuture<Response<String>> update(String modelUri, String updatedModel, String mediaType) {
+    public CompletableFuture<Response<String>> update(String modelUri, String updatedModel) {
```

Comment:
Could be improved by adding a parameter for the desired format. This way encoding and request URL modifications could be done internally. Should be more error-prone as well.

Reply:
Right, I've created https://github.com/eclipsesource/modelserver/issues/40 as a follow-up